src/main/java/ch/jalu/configme/properties/convertresult/ValueWithComments.java

```diff
@@ -0,0 +1,54 @@
+package ch.jalu.configme.properties.convertresult;
+
+import java.util.List;
+
+/**
+ * Wraps a value and allows to associate it with comments. Can be used as return type from
+ * {@link ch.jalu.configme.properties.Property#toExportValue}.
+ * <p>
+ * Prefer defining comments in {@link ch.jalu.configme.SettingsHolder} classes whenever possible.
```

Comment:
Describe a bit more "prefer"—mention that whenever comments can be defined _statically_, `SettingsHolder` should be preferred.